addons/mass_mailing/models/mass_mailing.py

```diff
@@ -451,6 +451,7 @@ def default_get(self, fields):
         return res
 
     active = fields.Boolean(default=True)
+    subject = fields.Char('Subject', help='Subject of emails to send', required=True)
```

Comment:
Don't forget the migration tips, as we are adding a required field. I think populate it with the name is a good start ;)

Reply:
Seems right. Copy name into subject for existing mailings seem correct.